src/layout/DwindleLayout.cpp

```diff
@@ -349,32 +349,21 @@ void CHyprDwindleLayout::onWindowCreatedTiling(CWindow* pWindow) {
 
     // let user select position -> top, right, bottom, left
     if (focusDirection != OneTimeFocus::NOFOCUS) {
-        switch (focusDirection) {
-            case OneTimeFocus::UP: {
-                verticalOverride       = true;
-                NEWPARENT->children[1] = OPENINGON;
-                NEWPARENT->children[0] = PNODE;
-                break;
-            }
-            case OneTimeFocus::RIGHT: {
-                horizontalOverride     = true;
-                NEWPARENT->children[0] = OPENINGON;
-                NEWPARENT->children[1] = PNODE;
-                break;
-            }
-            case OneTimeFocus::DOWN: {
-                verticalOverride       = true;
-                NEWPARENT->children[0] = OPENINGON;
-                NEWPARENT->children[1] = PNODE;
-                break;
-            }
-            case OneTimeFocus::LEFT: {
-                horizontalOverride     = true;
-                NEWPARENT->children[1] = OPENINGON;
-                NEWPARENT->children[0] = PNODE;
-                break;
-            }
-            case OneTimeFocus::NOFOCUS: break;
+
+        // this is horizontal
+        if (focusDirection % 2 == 0) {
+            verticalOverride = true;
+        } else {
+            horizontalOverride = true;
+        }
+
+        // 1,2 -> right and bottom 0 -> top and left
+        if (focusDirection % 3 == 0) {
```

Comment:
the logic here changed, it used to be UP & LEFT -> 1,0 and DOWN & RIGHT -> 0,1. Intentional?

Reply:
Oh sorry, this is the result of the modulo. I can specify this a bit better.
top: 0 % 3 -> 0
left : 3 % 3 -> 0
right: 1 % 3 -> 2
bottom: 2 % 3 -> 1
So yes the change is intentional.